apps/src/applab/applab.js

```diff
@@ -1162,6 +1162,7 @@ Applab.execute = function () {
   var codeWhenRun;
   if (level.editCode) {
     codeWhenRun = studioApp.getCode();
+    Applab.currentExecutionLog = [];
```

Comment:
gotcha. One other question - it looks like `Applab.currentExecutionLog` is just a reference to `Applab.JSInterpreter.executionLog`. Would it make more sense to have JSInterpreter do the reset, i.e. do something like `Applab.JSInterpreter.clearLog()` so that JSInterpreter is always the one mutating this state?